components/desktop/librecad/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Carsten Grzemba cgrzemba@opencsw.org
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME =	LibreCAD
+COMPONENT_VERSION =	2.1.3
+COMPONENT_PROJECT_URL =	https://github.com/LibreCAD
+COMPONENT_FMRI =		desktop/librecad
+COMPONENT_SRC =		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE =	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH =	\
+	sha256:5d0bc37efa1006742785c06ec30737c7866b0366048553c5560e06a3e442f7b3
+COMPONENT_ARCHIVE_URL=  https://api.github.com/repos/$(COMPONENT_NAME)/$(COMPONENT_NAME)/tarball/$(COMPONENT_VERSION)
+
+UNPACK_ARGS = --relocate-to=$(COMPONENT_SRC)
+COMPONENT_POST_UNPACK_ACTION = \
+	$(MV) $(COMPONENT_SRC)/*/* $(COMPONENT_SRC) ;\
+	$(MV) $(COMPONENT_SRC)/*/.[gt]* $(COMPONENT_SRC) ;\
+	rmdir $(COMPONENT_SRC)/$(COMPONENT_NAME)-$(COMPONENT_NAME)-[0-9a-f]*
+
+COMPONENT_LICENSE =	GPLv2
```

Comment:
If there are multiple licenses, add multiple license actions to p5m files and you can leave COMPONENT_LICENSE and COMPONENT_LICENSE_FILE empty,.